bootstrap/eks/controllers/eksconfig_controller.go

```diff
@@ -65,7 +66,8 @@ type EKSConfigReconciler struct {
 // +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;create;update;delete;
 
 func (r *EKSConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, rerr error) {
-	log := logger.FromContext(ctx)
+	ctx, log, done := tele.StartSpanWithLogger(ctx, "bootstrap.eks.controllers.eksconfig.Reconcile")
```

Comment:
It would probably be worth us attaching the `reconcileID` coming from controller-runtime.

This can be done with the help of `controller.ReconcileIDFromcontext(ctx)`

Reply:
Same applies for other spans we start within the `Reconcile` functions

Reply:
I also attached name and namespace from the request.